### PR TITLE
AV-223474:  Add gateway prefix to default backend HTTPPS

### PR DESCRIPTION
**Pull request by ashishnayak96**

### CODE REVIEW

**Comment by ashishnayak96** [comment]
build ako

File: ako-gateway-api/status/gateway_status.go [hunk 1]
```diff
@@ -148,6 +148,10 @@ func (o *gateway) Update(key string, option status.StatusOptions) {
 	}
 
 	gatewaystatus := akogatewayapiobjects.GatewayApiLister().GetGatewayToGatewayStatusMapping(gw.Namespace + "/" + gw.Name)
+	if gatewaystatus == nil {
```
Comment (akshayhavile) [inline ako-gateway-api/status/gateway_status.go:151]: @ashishnayak96: This is being addressed as part of this PR: https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1593/files#diff-6ab3147012ebebb189722a3e384914e6570b9aacf9757a0ac17946cfac1ae952

**Comment by ashishnayak96** [comment]
build ako

File: ako-gateway-api/status/gateway_status.go (hunk 1 above)
Comment (ashishnayak96) [reply ako-gateway-api/status/gateway_status.go]: removed

**Comment by ashishnayak96** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Review by arihantg — APPROVED** [review]

**Comment by akshayhavile** [comment]
@ashishnayak96 Do we have any UTs to validate default HTTPS functionality?

**Comment by ashishnayak96** [comment]
> @ashishnayak96 Do we have any UTs to validate default HTTPS functionality?

Not a separate test but we do validate default backend. https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1502/files#diff-82fa738631445982ab5dc53c988b8a9c32bfee40c8fa1726300a11968fd4c9e5R165

**Pull request merged**